exercism-modern.el

```diff
@@ -245,13 +245,12 @@ Pass prefix BUFFER-PREFIX-ARG to prompt for a buffer instead."
                                            (blurb (alist-get 'blurb exercise))
                                            (difficulty (alist-get 'difficulty exercise))
                                            (is-unlocked (not (eq :json-false (alist-get 'is_unlocked exercise))))
-                                           (is-recommended (not (eq :json-false (alist-get 'is_recommended exercise))))
                                            (text-face (if is-unlocked 'default 'shadow))
-                                           (foreground (face-attribute (intern (format "exercism-%s-button" difficulty)) :foreground))
-                                           (background (face-attribute (intern (format "exercism-%s-button" difficulty)) :background)))
+                                           (foreground (face-attribute (intern (format "exercism-modern-%s-button" difficulty)) :foreground))
+                                           (background (face-attribute (intern (format "exercism-modern-%s-button" difficulty)) :background)))
                                       (list slug
                                             (vector (concat
-                                                     (propertize "  " 'face 'warning)
+                                                     (propertize "★  " 'face 'warning)
```

Comment:
How does this look with font scaling? Massive/small fonts?

Reply:
They scale with the font:

<img width="1354" alt="Screen Shot 2022-10-06 at 7 46 38 PM" src="https://user-images.githubusercontent.com/20887/194443187-0efd7c4d-5d1e-43be-bba8-21ace252d9bb.png">

The svg wasn't so lucky

Reply:
That's good, annoying about the svgs though.

Can you raise an issue for that?